Exception: 
Error:     [1004]
Command:   add_table
Item:      44 
Label:     
Item Type: mvAppItemType::mvTable
Message:   Incompatible child. Acceptable children include:	mvAppItemType::mvTooltip
mvAppItemType::mvTableRow
mvAppItemType::mvTableColumn


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "c:\Users\valer\Documents\progval\Magick-notes\main.py", line 371, in <module>
    add_table(bas__yes, number_tag, id_jssson, "text" )
  File "c:\Users\valer\Documents\progval\Magick-notes\main.py", line 304, in add_table
    dpg.add_separator()
  File "C:\Users\valer\AppData\Local\Programs\Python\Python311\Lib\site-packages\dearpygui\dearpygui.py", line 6593, in add_separator
    return internal_dpg.add_separator(label=label, user_data=user_data, use_internal_label=use_internal_label, tag=tag, indent=indent, parent=parent, before=before, show=show, pos=pos, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
SystemError: <built-in function add_separator> returned a result with an exception set
